static/src/stylesheets/layout/new-header/_menu-item.scss

```diff
@@ -119,20 +99,26 @@
 
     @include mq(desktop) {
         font-size: 16px;
+        letter-spacing: .02rem;
         line-height: 18px;
-        padding-bottom: $gs-baseline / 2;
+        padding-bottom: 9px;
         padding-left: 0;
-        padding-right: $gs-gutter / 2 - 1;
-        padding-top: $gs-baseline / 2;
+        padding-right: 0;
+        padding-top: $gs-baseline / 4;
     }
 
     &:hover,
     &:focus {
         text-decoration: none;
 
-        // TODO: maybe unify?
         @include mq(desktop) {
-            color: #ffffff;
+            color: #00d4ff;
```

Comment:
We need a colour that is visually brighter/louder than the white, and none of our blues are bright enough 😛